Resources/Views/Submissions/Fields/textarea-input.leaf

```diff
@@ -0,0 +1,20 @@
+<div class="form-group" name="#(key)">
```

Comment:
Should we leave out the name on the form group to avoid having two elements with the same name? We also don't set this for the input fields.